### ISSUE TITLE
Do cool stuff with `team <-> project`

### ISSUE THREAD

**Issue by F3PiX** [rails-girls-summer-of-code/rgsoc-teams#818]
Blocked by #816 
Once the production db is updated with the `team.project_id`, we can use the team <->  project association for things like:

- [ ] 1 Orga Team edit page: assign project with dropdown select
- [ ] 2 Use the association in the query Projects#index
- [ ] 3 Turn the project_name in `team#display_name` into a link to the project show view.
- [ ] 4 Change relevant references to the `team.project_name` into `team.project.name`  
... ? << Add your Wishlist here 

Maybe split it up in separate PR's.

**Comment by mkalininait** [comment]
In addition to 1:
I think we should remove the `project_name` from the non-orga team edit page.

And this comment, too: https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues/19#issuecomment-322031515
